### PR TITLE
OHRM5X-1586: Fix recruitment - external candidate form related issues

### PR DESCRIPTION
**Pull request by Super-Chama**

### CODE REVIEW

**Review by RajithaKumara — APPROVED** [review]

File: src/client/src/orangehrmRecruitmentPlugin/pages/ApplyJobVacancy.vue [hunk 1]
```diff
@@ -19,184 +19,167 @@
...
+          </oxd-form-actions>
+        </div>
+      </oxd-form>
     </div>
   </div>
+  <div class="orangehrm-paper-container">
+    <oxd-text tag="p" class="orangehrm-vacancy-list-poweredby">
+      {{ $t('recruitment.powered_by') }}
+    </oxd-text>
+    <img
```
[the remark this answers is not in the document]
Comment (Super-Chama) [reply src/client/src/orangehrmRecruitmentPlugin/pages/ApplyJobVacancy.vue:188]: yes. default banner logo seems not transparent even if it is png. i'll check with Kavinda

File: src/client/src/orangehrmRecruitmentPlugin/pages/ApplyJobVacancy.vue [hunk 2]
```diff
@@ -19,184 +19,167 @@
...
+      <oxd-text class="orangehrm-vacancy-description" tag="p">
+        {{ $t('general.description') }}
+      </oxd-text>
+      <oxd-text
+        v-show="vacancyDescription"
+        class="orangehrm-vacancy-description"
+        tag="p"
+      >
+        <pre class="orangehrm-applicant-card-pre-tag"
+          >{{ vacancyDescription }}
```
[the remark this answers is not in the document]
Comment (Super-Chama) [reply src/client/src/orangehrmRecruitmentPlugin/pages/ApplyJobVacancy.vue]: this make some extra space in the description. kept as is

File: src/client/src/orangehrmRecruitmentPlugin/pages/public-job-vacancy.scss [hunk 3]
```diff
@@ -99,16 +100,24 @@
     &-header {
       display: flex;
       justify-content: space-between;
-      flex-direction: column-reverse;;
+      flex-direction: column-reverse;
```
Comment (RajithaKumara) [inline src/client/src/orangehrmRecruitmentPlugin/pages/public-job-vacancy.scss:103]: :relaxed:

**Review by RajithaKumara — APPROVED** [review]

File: src/client/src/orangehrmRecruitmentPlugin/pages/public-job-vacancy.scss (hunk 3 above)
Comment (Super-Chama) [reply src/client/src/orangehrmRecruitmentPlugin/pages/public-job-vacancy.scss:103]: missed it :angel:

**Pull request merged**